src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3455,9 +3463,15 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
             // Use at most 3 locks (we don't expect contention on that many cores anyways,
             // and typically we'll be processing just 2 or 3 files anyway, hence capacity=3.
             ConcurrentDictionary<string, string> filesProcessed
-                = new ConcurrentDictionary<string, string>(/*concurrencyLevel*/3, /*capacity*/3, StringComparer.OrdinalIgnoreCase);
-            Parallel.ForEach(Types, sste =>
-            {
+                = new ConcurrentDictionary<string, string>(
```

Comment:
Seems like this line should be part of line 3457.  It was probably split previously because it was long.

Reply:
Done.